source/_integrations/reddit.markdown

```diff
@@ -14,6 +14,10 @@ The Reddit sensor integrates data from [Reddit](https://reddit.com/) to monitor
 
 To set up this sensor, you will need to generate a `client_id` and `client_secret` for the user account you will use to connect. Follow the first steps in [this Wiki page](https://github.com/reddit-archive/reddit/wiki/OAuth2-Quick-Start-Example).
 
+<div class='note'>
+This integration does not support Reddit two-factor authentication. If you use two-factor authentication for your Reddit account, create a separate Reddit account without two-factor authentication for use with homeassistant.
```

Comment:
> This integration does not support Reddit's two-factor authentication. If you use two-factor authentication for your Reddit account, create a separate Reddit account without two-factor authentication for use with Home Assistant.

Reply:
Hi @frenck, I agree with your suggested changes, but i can't figure out how to make those changes since i've already deleted my fork of home-assistant.io...

Reply:
In that case, you can't 🤷‍♂ 

The best thing to do right now, I guess, is to open up a new PR. With a small change like this, it would be a tiny copy & paste action. 

Thanks already! 👍